server/utils/index.js

```diff
@@ -356,20 +404,16 @@ export const generateMetaComponents = ({
 		outputComponents = [
 			...outputComponents,
 			<meta key="pa1" property="article:published_time" content={publishedAt} />,
-			<meta
-				key="pa2"
-				property="citation_publication_date"
-				content={googleScholarPublishedAt}
-			/>,
-			<meta key="pub1" property="citation_publisher" content="PubPub" />,
+			<meta key="pa2" name="citation_publication_date" content={googleScholarPublishedAt} />,
+			<meta key="pub1" name="citation_publisher" content="PubPub" />,
 			<meta key="pub2" property="dc.publisher" content="PubPub" />,
 		];
 	}
 
 	if (doi) {
 		outputComponents = [
 			...outputComponents,
-			<meta key="doi1" property="citation_doi" content={`doi:${doi}`} />,
+			<meta key="doi1" name="citation_doi" content={`doi:${doi}`} />,
```

Comment:
Out of curiosity, any particular reason this one is `name` while the others are `property`?

Reply:
I found out that, technically, all of the Highwire/Google Scholar (`citation_`) tags should be `name`, while all of the Dublin Core (`dc:`) tags should be `property`. Zotero was failing to grab some of the tags because of that (I [suggested](https://forums.zotero.org/discussion/80424/zotero-not-pulling-in-a-few-highwire-fields-for-pubpub#latest) they parse both fields like some other platforms do).

So, all the `citation_` tags should now be name, all of the `dc:` tags should now be `property`.